package.json

```diff
@@ -40,5 +51,42 @@
   },
   "dependencies": {
     "mobx": "^5.9.0"
+  },
+  "config": {
+    "commitizen": {
+      "path": "./node_modules/cz-conventional-changelog"
+    }
+  },
+  "commitlint": {
+    "extends": [
+      "@commitlint/config-conventional"
+    ]
+  },
+  "husky": {
+    "hooks": {
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
+    }
+  },
+  "release": {
+    "branch": "chore/sl-83/prism-packaging-and-binary-deployment",
+    "verifyConditions": [
+      "@semantic-release/git",
+      "@semantic-release/github",
+      "@semantic-release/npm"
+    ],
+    "prepare": [
+      "@semantic-release/git",
```

Comment:
Remove. in fact, not sure we need `prepare`, `publish`, `success`, or `fail`?

Reply:
Oh yeah I have that commit locally but forgot to push :) one second!